server/src/services/token.service.ts

```diff
@@ -1,13 +1,13 @@
-import jwt from "jsonwebtoken";
+import jwt, { JwtPayload } from "jsonwebtoken";
 import ApiError from "../error/api.error.js";
 
 export default class TokenService {
-  generateTokens(payload) {
+  generateTokens(payload: object) {
     try {
-      const accessToken = jwt.sign(payload, process.env.JWT_ACCESS_SECRET, {
+      const accessToken = jwt.sign(payload, process.env.JWT_ACCESS_SECRET!, {
         expiresIn: "30m",
       });
-      const refreshToken = jwt.sign(payload, process.env.JWT_REFRESH_SECRET, {
+      const refreshToken = jwt.sign(payload, process.env.JWT_REFRESH_SECRET!, {
```

Comment:
for what do you need '!'?

Reply:
so I promise that the object is not undefined so that I can pass it